lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -152,12 +164,31 @@ class VersionSectionSelector extends Component<SelectorProps> {
         }
       })
     }
+
     const text = hasCriticalError
       ? 'critical error'
       : validationSummary.errorCount
+
+    let diffLabel
+    if (comparedVersion && comparedVersionIndex) {
+      const diff = validationSummary.errorCount - comparedVersion.validationSummary.errorCount
+      const diffColor = diff > 0 ? 'label-danger' : (diff < 0 ? 'label-success' : 'label-default')
```

Comment:
Please rename variable to `diffClassName`

Reply:
This was addressed in https://github.com/ibi-group/datatools-ui/pull/528/commits/5fbbb2e918cdcaa58e793013574d7b7d5d0ae0d6.